client/src/routes/login/tabs/SignIn.tsx

```diff
@@ -30,6 +46,35 @@ function SignIn() {
 
 	const signIn = (e: React.FormEvent<HTMLFormElement>) => {
 		e.preventDefault();
+
+		(async () => {
+
+			const params = buildRequestParams({
+				email: email,
+				password: password,
+				persistentLogin: rememberMe,
+			});
+
+			const request = await fetch(`api/signin?${params}`, {
```

Comment:
zelfde verhaal met dit gebruik de API methode. en zet de user params niet in de uri, als een aanvaller dat ziet dan krijgt hij de informatie van de user, zet dat liever in de body